tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -9,7 +9,8 @@ load "${BATS_TEST_DIRNAME}/lib.sh"
 load "${BATS_TEST_DIRNAME}/confidential_common.sh"
 
 setup() {
-    confidential_setup || skip "Test not supported for ${KATA_HYPERVISOR}."
+    confidential_setup && skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
```

Comment:
This skips the non-TEE tests as well, which I don't think we want to do?

Reply:
Sorry, that was missed from my side, as I didn't realise that the `confidential_setup` check would actually check for non-TEEs as well. :-/

I will raise a follow-up.